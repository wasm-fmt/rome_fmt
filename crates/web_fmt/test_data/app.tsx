export function App<T extends number>(): JSX.Element {
return <div>Hello, world!</div>;
}