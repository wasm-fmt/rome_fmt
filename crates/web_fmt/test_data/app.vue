<template>
  <div>
    <div v-for="item in items" :key="item.id">
      <div v-if="item.type === 'text'">
        <p>{{ item.text }}</p>
      </div>
      <div v-else-if="item.type === 'image'">
        <img :src="item.src" :alt="item.alt" />
      </div>
      <div v-else-if="item.type === 'video'">
        <video :src="item.src" :alt="item.alt" />
      </div>
    </div>
  </div>
</template>
<script>
export default {
  props: {
    items: {
      type: Array,
      required: true
    }
  }
}
</script>
<style lang="scss" scoped>
    img, video {
        max-width: 100%;
    }
</style>
