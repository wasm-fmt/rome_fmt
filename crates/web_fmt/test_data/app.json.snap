{
	"hello": "world",
	"foo": "bar",
	"baz": [
		"qux",
		"quux",
		{
			"corge": "grault",
			"garply": "waldo"
		}
	]
}
